clients/drcachesim/tracer/tracer.cpp

```diff
@@ -974,12 +968,11 @@ memtrace(void *drcontext, bool skip_size_cap)
                 if (have_phys && op_use_physical.get_value() &&
                     type != TRACE_TYPE_THREAD && type != TRACE_TYPE_THREAD_EXIT &&
                     type != TRACE_TYPE_PID) {
-                    addr_t virt = instru->get_entry_addr(mem_ref);
-                    addr_t phys = data->physaddr.virtual2physical(virt);
                     DR_ASSERT(type != TRACE_TYPE_INSTR_BUNDLE);
-                    if (phys != 0)
```

Comment:
We should document at `virtual2physical` that zero is a valid translation.

Reply:
Done.  Also mentioned high values due to holes.